.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -0,0 +1,71 @@
+---
+name: "🥇 New LTS release checklist"
+labels: lts
+about: Track work required for a new LTS release
+---
+
+# Next LTS release
+
+## Release Lead
+
+<!-- 
+The release lead is the person who makes sure that all steps are completed
+Not necessarily the person doing all the work
+
+This role should rotate between LTS releases
+-->
+
+@<github-username of release lead>
```

Comment:
I love this idea, to know who's responsible

Reply:
Is it worth having a separate assignment of someone that monitors the results of the build?

As an example, I've recently been monitoring LTS and weekly release execution with Oliver as the release lead.  He created the initial pull request, walked it through the review process, and merged it.  He then relied on others to confirm the final build and packaging process was successful.  

Would that work or is the assumption that the release lead watches things all the way to completion of the release?

Reply:
I had the release lead as the person who was making sure everything got done, not necessarily doing the things, anyone can put their name next to a task / put their hand up to do part of the release